prow/plugins/help/help.go

```diff
@@ -30,18 +30,31 @@ import (
 const pluginName = "help"
 
 var (
-	helpLabel         = "help wanted"
-	helpRe            = regexp.MustCompile(`(?mi)^/help\s*$`)
-	helpRemoveRe      = regexp.MustCompile(`(?mi)^/remove-help\s*$`)
-	helpGuidelinesURL = "https://git.k8s.io/community/contributors/devel/help-wanted.md"
```

Comment:
Reminder that prow is used by other communities than kubernetes. 

I think this is fine (they can send us a PR if they want to change it and/or maybe we like giving them a good default).